corehq/util/log.py

```diff
@@ -131,6 +131,18 @@ def get_code(self, extracted_tb):
         )
         )
 
+    @classmethod
+    def _clean_subject(cls, subject):
+        # Django raises BadHeaderError if subject contains following bad_strings
+        # to guard against Header Inejction.
+        # see https://docs.djangoproject.com/en/1.8/topics/email/#preventing-header-injection
+        # bad-strings list from http://nyphp.org/phundamentals/8_Preventing-Email-Header-Injection
+        bad_strings = ["\r", "\n", "%0a", "%0d", "Content-Type:", "bcc:", "to:", "cc:"]
+        replacement = "-"
```

Comment:
subjective but i'd use a space character - especially if newlines are the most common thing